include/flox/package.hh

```diff
@@ -170,42 +170,47 @@ public:
   virtual nlohmann::json
   getInfo( bool withDescription = false ) const
   {
-    std::string                system = this->getPathStrs().at( 1 );
-    nlohmann::json             j      = { { system,
-                                            { { "name", this->getFullName() },
-                                              { "pname", this->getPname() } } } };
-    std::optional<std::string> os     = this->getVersion();
+    System system = this->getPathStrs().at( 1 );
 
-    if ( os.has_value() ) { j[system].emplace( "version", *os ); }
-    else { j[system].emplace( "version", nlohmann::json() ); }
+    nlohmann::json jto = { { system,
+                             { { "name", this->getFullName() },
+                               { "pname", this->getPname() } } } };
 
-    os = this->getSemver();
-    if ( os.has_value() ) { j[system].emplace( "semver", *os ); }
-    else { j[system].emplace( "semver", nlohmann::json() ); }
+    std::optional<std::string> oos = this->getVersion();
```

Comment:
Nitpick: These really short variable names are hard to read. I have no idea what `oos` means.

Reply:
It was just `os` before ;)
I made it 3 characters to get `clang-tidy` to stop griping.